Examples/Swift/FillPatternExample.swift

```diff
@@ -0,0 +1,86 @@
+import Mapbox
+
+@objc(FillPatternExample_Swift)
+
+class FillPattern_Swift: UIViewController, MGLMapViewDelegate {
+    var fillPatternImage: UIImage!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let mapView = MGLMapView(frame: view.bounds)
+        
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.styleURL = MGLStyle.darkStyleURL(withVersion: MGLStyleDefaultVersion)
```

Comment:
This example uses a named layer in the dark style, so it should be pinned to the specific version of that style and not the generic default style version (which can be updated with any release).